python/lsst/cp/pipe/ptc/cpExtractPtcTask.py

```diff
@@ -417,7 +417,7 @@ def run(self, inputExp, inputDims, taskMetadata, inputPhotodiodeData=None):
         # used to ensure that the number of output and input
         # dimensions match.
         dummyPtcDataset = PhotonTransferCurveDataset(ampNames, 'DUMMY',
-                                                     self.config.maximumRangeCovariancesAstier)
+                                                     self.config.maximumRangeCovariancesAstier, 1)
```

Comment:
I added the keyword for the measured matrix size, and deleted the fit size, since now the default behavior takes care of it (if it's `None`, the measured one is assigned).